### ISSUE TITLE
RStudio on Windows attempts to parse file as R after edit

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#4588]
### System details

    RStudio Edition : Desktop
    RStudio Version : v1.2.1335
    OS Version      : macOS 10.14.4
    R Version       : 3.5.3

### Steps to reproduce the problem

Run this code:

```
utils::file.edit("hello.txt")
```

and then write the contents:

> Hello, world!

and save the file.

### Describe the problem in detail

After save, RStudio complains:

```
> utils::file.edit("hello.txt")
Error in editor(file = file, title = title) : 
  Error in parse(targetFile) : hello.txt:1:6: unexpected ','
1: Hello,
         ^
You can attempt to correct the error using hello.txt = edit()
```

### Describe the behavior you expected

RStudio shouldn't attempt to parse files saved after edit as R code.

Note that this is normally done to support the `edit()` command, e.g.

```
edit(rnorm)
```

**Comment by kevinushey** [comment]
Marking as v1.2-patch as this affects renv.

**Comment by kevinushey** [comment]
This can be worked around in `renv`; we can leave the fix for RStudio v1.3.

**Comment by dfalty** [comment]
Verified fixed 1.3.867
